node/src/Commands.ts

```diff
@@ -1056,6 +1056,23 @@ export function createLIndex(
     return createCommand(RequestType.LIndex, [key, index.toString()]);
 }
 
+export enum InsertPosition {
```

Comment:
Do we need a doc for enum (not for enum variants)?

Reply:
Added docs, we have docs for the enum variants in java and elsewhere in the Commands.ts file so I added docs for them as well